app/presenters/claim_csv_presenter.rb

```diff
@@ -1,40 +1,12 @@
 class ClaimCsvPresenter < BasePresenter
```

Comment:
Also, again, not you, but this class seems to only be used by the ManagementInformationGenerator for generating the MI report. So its name would be clearer as ManagementInformationPresenter

Reply:
Have renamed ClaimCsvPresenter to ManagementInformationPresenter